gkr_iop/src/gkr/layer/cpu/mod.rs

```diff
@@ -302,7 +302,7 @@ pub(crate) fn prove_rotation<E: ExtensionField, PCS: PolynomialCommitmentScheme<
     )
     .collect_vec();
     exit_span!(span);
-    // TODO FIXME: we pick a random point from output point, does it sound?
+    // TODO TODO: we pick a random point from output point, does it sound?
```

Comment:
nitpick: some leftover due to replacement I guess

Reply:
Ops... Thanks